.gitmodules

```diff
@@ -119,6 +119,6 @@
     path = android/libs/fbjni
     url = https://github.com/facebookincubator/fbjni.git
 [submodule "third_party/XNNPACK"]
+    ignore = dirty
```

Comment:
What is this?

Reply:
From my understanding is that, it suppresses listing of dirty submodule entries in git status. I did not know till this morning. Just googled and found this: https://stackoverflow.com/questions/3240881/git-can-i-suppress-listing-of-modified-content-dirty-submodule-entries-in-sta.